File: Final-Project-Front/src/app/models/LoanStateDTO.ts
import {LOAN_STATE} from "./LOAN_STATE";

export class LoanStateDTO{


  constructor(
    private _loanState: LOAN_STATE
  ) {

  }


  get loanState(): LOAN_STATE {
    return this._loanState;
  }

  set loanState(value: LOAN_STATE) {
    this._loanState = value;
  }

  public toJSON(): any {
    return {
      loanState: this.loanState,
    };
  }
}

File: Final-Project-Back/edge-service/src/main/java/com/ironhack/edgeservice/controller/dto/LoanStateDTO.java
package com.ironhack.edgeservice.controller.dto;

import com.ironhack.edgeservice.enums.LOAN_STATE;

public class LoanStateDTO {

    private LOAN_STATE LOANSTATE;

    public LoanStateDTO() {
    }

    public LoanStateDTO(LOAN_STATE LOANSTATE) {
        this.LOANSTATE = LOANSTATE;
    }

    public LOAN_STATE getLoanState() {
        return LOANSTATE;
    }

    public void setLoanState(LOAN_STATE LOANSTATE) {
        this.LOANSTATE = LOANSTATE;
    }
}
